docs/development/maintenance.md

```diff
@@ -1,8 +1,10 @@
 ILIAS Maintenance
 =================
-The development of the ILIAS source code is coordinated and maintained by a coordination team within the ILIAS 
-network. Besides the main responsibilities for the project, several developers and users are maintaining certain 
-modules of ILIAS.
+The development of the ILIAS is coordinated by the Product Manager and the
```

Comment:
Das "of THE ILIAS" liest sich für mich persönlich etwas merkwürdig? Warum nicht einfach "of ILIAS"?